src/docs/product/releases/index.mdx

```diff
@@ -352,3 +352,31 @@ You can also use our [API](/api/releases/create-a-new-deploy-for-an-organization
 JavaScript and React Native projects can utilize release artifacts to unminify or symbolicate error stack traces, you can find the artifacts uploaded to Sentry by navigating to **[Project] » Project Settings » Source Maps**.
 
 To learn more, please check out our [Source Maps](/platforms/javascript/sourcemaps/) docs.
+
+## Archiving Releases
+
+Since the received event from the deleted release would automatically recreate it, Sentry has an option to hide releases permanently from the UI. Archiving releases can be useful when you're no longer interested in seeing them or you've accidentally created a release you no longer want.
```

Comment:
```suggestion
Since an event received from a deleted release automatically recreates it, Sentry has an option to hide releases permanently from sentry.io. In addition, archiving releases can be useful if you're no longer interested in viewing the release or if you've accidentally created a release you no longer want to track.
```